### ISSUE TITLE
Load Feature options the same way as normal options

### ISSUE THREAD

**Issue by DGuhr** [keycloak/keycloak#9549]
### Description

At the moment, Configuration / CLI options are loaded into the cli / into the docs from 2 places. "Normal" options, which map to 1 or more quarkus properties, are loaded via `configuration/mappers/<category>mappers.java` classes, but the Feature-related buildtime options are loaded from Profile  directly and added to picocli, see [here](https://github.com/keycloak/keycloak/blob/4c747047cef126a182b78cb1238824b447c1fbc9/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java#L303-L332). 

We should unify our approach, so that the mapper-classes and package are the "one place to go to" to add new configuration options. This also reduces the maintenance overhead, e.g. for the new guides (which lack features atm, because they are not loaded by `PropertyMappers.getMappers()`).

So basically, having a 'FeaturesPropertyMapper.java' Class in [the mappers directory
](https://github.com/keycloak/keycloak/tree/main/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers) and adding these to [PropertyMappers.java](https://github.com/keycloak/keycloak/blob/main/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java#L24-L32) like all the other Mappers. This would allow to work with all options in a unified way, e.g. when getting the options for the guides templates like it's done [here](https://github.com/keycloak/keycloak/blob/4c747047cef126a182b78cb1238824b447c1fbc9/docs/maven-plugin/src/main/java/org/keycloak/guides/maven/Options.java#L17-L20).

**Comment by stianst** [comment]
Closing this as it is no longer relevant when we don't have per-feature options
